examples/capi-quick-start/docker-cluster-cilium-helm-addon.yaml

```diff
@@ -3,33 +3,33 @@ kind: Cluster
 metadata:
   labels:
     cluster.x-k8s.io/provider: docker
-  name: docker-quick-start-helm-addon-cilium
+  name: ${CLUSTER_NAME}
 spec:
   clusterNetwork:
     pods:
       cidrBlocks:
-        - 192.168.0.0/16
+        - ${POD_CIDR:-192.168.0.0/16}
     serviceDomain: cluster.local
     services:
       cidrBlocks:
-        - 10.128.0.0/12
+        - ${SERVICE_CIDR:-10.128.0.0/12}
   topology:
     class: docker-quick-start
     controlPlane:
       metadata: {}
-      replicas: 1
+      replicas: ${CONTROL_PLANE_MACHINE_COUNT}
     variables:
       - name: clusterConfig
         value:
           addons:
             cni:
-              provider: cilium
+              provider: Cilium
```

Comment:
Just checking the capitalization does not matter here, it will work either way right?
(just thinking about the downstream projects)

Reply:
It does matter.
```
The Cluster "docker-quick-start-helm-addon-cilium" is invalid: spec.topology.variables.addons.cni.provider: Unsupported value: "Cilium": supported values: "calico", "cilium"
```

Reply:
Since `strategy: HelmAddon` is already capitalized, should we make these capitalized too?  
🤔 but there is also other addons `"aws"` `"aws-ebs"`.

Im good either way.

Reply:
Doh sorry i made a mistake not pulling in the case change... Yes I wanted to make it capitalised. Let me do that now.

Reply:
Fixed now. This would have been caught in e2e tests in #360 😉